client/galaxy/style/less/base.less

```diff
@@ -1719,3 +1719,23 @@ div.toolTitleNoSection
 #for_bears {
     display: none;
 }
+
+// communication channel bootstrap modal 
```

Comment:
We should move this as well into the plugin directory or at least make it communication independent.